src/commands/generate_config.yml

```diff
@@ -9,30 +9,6 @@ parameters:
 steps:
   - run:
       name: Generate configuration
-      command: |
-        cat << parameters.pipeline_parameters_json_file >>
-
-        echo "version: 2.1" > /tmp/generated-config.yml
-
-        lambda=$(jq -r '.lambda' << parameters.pipeline_parameters_json_file >>)
-        if [ $lambda == true ]; then
-          echo "Lambda deployment triggered"
-          if [ ! -f .circleci/serverless.yml ]; then
-            echo "Config .circleci/serverless.yml not found"
-            exit 1
-          fi
-          yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' /tmp/generated-config.yml .circleci/serverless.yml
-        fi
-
-        tf=$(jq -r '.tf' << parameters.pipeline_parameters_json_file >>)
-        if [ $tf == true ]; then
-          echo "Terraform deployment triggered"
-          git clone --depth 1 --no-checkout git@github.com:FigurePOS/circle-ci-node-ecs-orb.git
-          cd circle-ci-node-ecs-orb
-          git sparse-checkout set workflows
-          git checkout master
-          cd ..
-          yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' /tmp/generated-config.yml circle-ci-node-ecs-orb/workflows/tf.yml
-        fi
-
-        yq eval-all --inplace 'select(fileIndex == 0) * select(fileIndex == 1)' /tmp/generated-config.yml .circleci/app.yml
+      environment:
+        CONFIG_JSON_FILE: << parameters.pipeline_parameters_json_file >>
+      command: << include(scripts/generate-config.sh) >>
```

Comment:
To jsem jen dal do samostatnýho souboru (`scripts/generate-config.sh`), líp se s tím pak pracuje v IDE.